Rakefile

```diff
@@ -1,7 +1,25 @@
+require 'data_mapper'
```

Comment:
Still don't really understand the need for rake. Are there any good resources online that would help me better understand it?